web/client/epics/map.js

```diff
@@ -178,7 +183,20 @@ const zoomToExtentEpic = (action$, {getState = () => {} }) =>
         }
         return legacyZoomToExtent({...action, extent}, mapSelector(getState()) );
     });
+/**
+ * It checks user's permissions on current map on LOGIN_SUCCESS event
+ * @memberof epics.map
+ * @param {object} action$
+ */
+const checkMapPermissions = (action$, {getState = () => {} }) =>
+        action$.ofType(LOGIN_SUCCESS)
+        .map(() => {
+            const mapId = mapIdSelector(getState());
+            return loadMapInfo(ConfigUtils.getConfigProp("geoStoreUrl") + "extjs/resource/" + mapId, mapId);
```

Comment:
Triggering a thunk in an epic do not work as far as I know. `loadMapInfo` is a thunk. 
Are you sure this works?

Reply:
Since this f1f9a271ebce3c1719ee94ef1fc0d57166f8139f thunkmiddleware is applied before redux-observable and so It's possible to dispatch from an epic. [https://stackoverflow.com/questions/48510274/use-of-redux-thunk-with-redux-observable](url)

Reply:
Ok I verified that this issue was present during the first 6 months we used redux-observable, but due to the refactor, this usage is allowed. So I think it can be approved.